app/experimenter/experiments/views.py

```diff
@@ -238,6 +245,12 @@ def in_qa_filter(self, queryset, name, value):
         else:
             return queryset
 
+    def surveys_filter(self, queryset, name, value):
+        if value:
+            return queryset.filter(survey=True)
+        else:
```

Comment:
nit: don't need the else here

Reply:
I think you do.. I tried to take it away and I got errors. I think every filter has to return a queryset by default.

Reply:
No you literally don't need the `else`:

    if value:
        return queryset.filter(...)
    return queryset